frontend/src/components/Comment/CommentList.js

```diff
@@ -1,43 +1,36 @@
 import React from 'react'
-import Comment from './Comment'
-import { getCommentByPicture } from './CommentAPI';
+import { useState, useEffect } from 'react'
```

Comment:
I understand the need for this import but also, through the declaration line `import React from 'react'` you technically already have access to these hooks. All you would have to do is call `React.useState` and `React.useEffect`.